tests/maskers/test_text.py

```diff
@@ -140,19 +147,25 @@ def test_text_infill_with_collapse_mask_token():
     mask_ex4 = np.array([False, False, False, False, False, False, False, False, False])
 
     text_infilled_ex1 = masker(mask_ex1, s)[0][0]
+    text_infilled_ex1_mist = masker_mistral(np.append(True, mask_ex1), s)[0][0]
```

Comment:
Just for me to clarify: we need to append True from the left because mistralai has `self.keep_prefix == 1` correct?

Reply:
Yes, exactly. In the text masker special tokens are replaced by "", which is why it does not appear in the output.